### PR TITLE
Fix regression with field trip date display

### PR DESCRIPTION
**Pull request by demory**
Fixes bug where clicking on a field trip entry in the FT request list (see screenshot) caused browser crash due to date library issue. Should now work when selecting requests both with date set (e.g. requests 35-37) and not set (e.g. request 34).

image

### CODE REVIEW

File: lib/components/admin/field-trip-details.js [hunk 1]
```diff
@@ -167,7 +166,11 @@ class FieldTripDetails extends Component {
               }}
               type="date"
             />
-            ({formatDistanceToNow(travelDate, { addSuffix: true })})
+            {travelDateValue && (
+              <span>
```
Comment (binh-dam-ibigroup) [inline lib/components/admin/field-trip-details.js:170]: Do we need this extra `<span>`?

**Review by binh-dam-ibigroup — APPROVED** [review]
One small nitpick, but I'm good with merging as is.

File: lib/components/admin/field-trip-details.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { format, formatDistanceToNow, parse } from 'date-fns'
 import {
   getDateFormat,
   // yyyy-MM-dd, which is same format required for date input controls.
+  OTP_API_DATE_FORMAT,
   OTP_API_DATE_FORMAT_DATE_FNS
```
Comment (amy-corson-ibigroup) [inline lib/components/admin/field-trip-details.js:12]: Out of curiosity, does `OTP_API_DATE_FORMAT_DATE_FNS` still need to be imported if it's been removed below?

**Review by amy-corson-ibigroup — APPROVED** [review]

File: lib/components/admin/field-trip-details.js (hunk 2 above)
Comment (binh-dam-ibigroup) [reply lib/components/admin/field-trip-details.js:12]: Good spot @amy-corson-ibigroup, thanks, that constant doesn't exist anymore so it can be removed.

**Pull request merged**